### DISCUSSION TITLE
[灌水] 下午的健身房

### DISCUSSION THREAD

**Discussion by ZhuYuan-148** [share121/inter-knot#1689] [其他]
![mmexport1723452337340](https://github.com/user-attachments/assets/a7edf16a-1fab-4ae1-bd9e-11c23e85c5bc)

本来说好要去空洞内巡查的，上级却突然又给了休假… 最近的休假未免有点太多了，感觉全身都变懒散了。所以和前辈一起去健身房举铁了，没想到最重的硬拉也才180kg，当即做了十组，运动的感觉真是好极了。

 只不过前辈一直在盯着我看…她说是在观察人类身体健身前后的变化，但这样被人直勾勾的盯着多少有点不太好受吧…，罢了罢了，我怎么能质疑前辈的话呢？

> 

> 

> 

> 图作者：pixiv` 氯酸钾的分解

**Comment by awen51299** [comment]
求原图啊哥哥

**Reply by ZhuYuan-148** [reply]
收到收到

**Comment by SSTV-10068** [comment]
180公斤...10组....就算早就知道前辈很厉害，但是没想到居然那么厉害！吓到小编了。

**Comment by jinuo12345** [comment]
厉害
